src/io/elf_loader.hpp

```
@@ -23,9 +23,9 @@ class MalformedElf final : public UnmarshalError {
 };
 
 int create_map_crab(const EbpfMapType& map_type, uint32_t key_size, uint32_t value_size, uint32_t max_entries,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)